functions/src/Services/Event.ts

```diff
@@ -1,12 +1,15 @@
-// const { App } = require('../app.js');
-// import * as App from 'firebase-admin';
 import { User } from "../Models/User";
 import { Event, eventConverter } from "../Models/Event";
 import { firestore } from 'firebase-admin';
 
```

Comment:
Maybe somewhere up here there could be some explanation as to what you should have on this file and what you shouldn't have on this file.